testing/internal/e2e/tests/static/target_address_test.go

```diff
@@ -0,0 +1,282 @@
+package static_test
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/boundary/testing/internal/e2e"
+	"github.com/hashicorp/boundary/testing/internal/e2e/boundary"
+	"github.com/stretchr/testify/require"
+)
+
+// TestCreateUpdateTargetAddress asserts that creating and updating a target
+// using a valid address is successful.
+func TestCreateUpdateTargetAddress(t *testing.T) {
```

Comment:
```suggestion
func TestCliCreateUpdateTargetAddress(t *testing.T) {
```

Can we add `Cli` to the names here to denote that these are tests for the CLI? Additionally, you could look into a similar test using the Go Api (without needing to test the `connect` pieces).